data/hai/hai missions.txt

```diff
@@ -1658,3 +1658,145 @@ mission "Strider notification"
 				decline
+
+
+
+mission "Station Exploration 1"
+	minor
+	name "Bring students to <planet>"
+	description "Bring a group of Hai and human students to <destination> to explore the abandoned station. They promise to pay you <payment>."
+	passengers 5
+	source "Greenwater"
+	destination "Darkrest"
+	to offer
+		random < 30
+		has "First Contact: Hai: offered"
+	on offer
+		conversation
+			`It is a sunny afternoon as you idly walk through the spaceport, until you somehow find yourself surrounded by a group of young humans and Hai.`
+			`	"You're a spaceship captain, right? Could you take us to <planet>?" says one of the Hai, as if he had practiced this line for a while now.`
+			`	The human next to him gives him a nudge. "Don't forget to ask for a trip back, Tagreep, unless you want to be stranded there."`
+			choice
+				`	"What would you want to do on <planet>? Isn't that an abandoned station?"`
+					goto question
+						to display
+							has "visited planet: Darkrest"
+				`	"Why do you want to go to <planet>?"`
+					goto "question 1"
+						to display
+							not "visited planet: Darkrest"
+				`	"Sure, I can take you there and back."`
+					goto accept
+				`	"How much can you pay me?"`
+				`	"Sorry, I'm busy right now. Go ask some other captain."`
+						decline
+			`	The group starts to shuffle around with a nervous air. "Well, we're still in school, so money's kind of tight right now... But we can pay you!" The Hai pauses to glance at his friends and, in almost a whisper, says, "is <payment> okay?"`
+			choice
+				`	"Why would you want to pay money to go to <planet>, anyhow? Isn't that an abandoned station?"`
+					to display
+						has "visited planet: Darkrest"
+				`	"Why do you want to go to <planet>?"`
+					goto "question 1"
+					to display
+						not "visited planet: Darkrest"
+				`	"Sure, I'm not hurting for cash right now."`
+					goto accept
+				`	"No way, that's too low. Find someone else."`
+					decline
+			label question
+			`	"Well, that's kind of why we want to go there," says another one of the Hai. "Some guy in our dorm told us that there's ghosts haunting <planet>, and we wanted to call his bluff."`
+				goto continue
+			label "question 1"
+			`	"Some guy in our dorm told us that there's ghosts haunting <planet>," says another one of the Hai, "and we wanted to call his bluff."`
+			label continue
+			`	"We're planning to look around for an afternoon, see what we can find," one of the human students says, grinning. "Once we're done we'll let you know, and then we can leave."`
+			`	One of the shorter Hai in the back pipes up. "I brought cameras for the trip!" He is ignored by the rest of the group.`
```

Comment:
RIP this lil guy.

Reply:
![image](https://github.com/endless-sky/endless-sky/assets/65418682/768b95a4-13e1-4816-94f1-0f2b69985e23)